source/probability_theory_3.Rmd

```diff
@@ -246,13 +250,11 @@ girl_counts = np.zeros(10000)
 # Do 10000 trials
 for i in range(10000):
 
-    # Generate 4 numbers at random, either "1" or "2." This is analogous to
-    # flipping a coin 4 times to generate 4 heads or tails. We keep these 
-    # numbers in a, letting "1" represent girls.
-    children = rnd.integers(1, 3, size=4)
+    # Select 'girl' or 'boy' at random, four times.
+    children = rnd.choice(['girl', 'boy'], size=4)
 
     # Count the number of girls and put the result in b.
-    b = np.sum(children == 1)
+    b = np.sum(children == 'girl')
```

Comment:
Much approved here, I definitely prefer using strings rather than numbers for simulations like this in a tutorial context, it takes away a level of abstraction.